### ISSUE TITLE
Script to convert Mask R-CNN model to ONNX

### ISSUE THREAD

**Issue by FraPochetti** [onnx/models#421]
Hi all, 
I am able to load and use your [ONNX Mask R-CNN model](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/mask-rcnn/model) following your very detailed [instructions](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/mask-rcnn), which is great.

Given that, I was wondering if you could provide more details around the script you used to convert the original trained model to ONNX.
I come from a PyTorch background and I am having hard times to get the official `torchvision` model exported correctly to ONNX, so I hoped you guys could share the code you used to do that (even if you didn't train the model in PyTorch in the first place).
It would be super helpful!

Thanks a lot

**Comment by rsomani95** [comment]
+1 for this.

Converting from PyTorch is possible but inference results don't match up exactly with Mask-RCNN (even with a batch size of 1), and it's hard to find some solid reference code for the export, so this would indeed be much appreciated!

**Comment by ebarsoum** [comment]
@spandantiwari

**Comment by pra-dan** [comment]
@FraPochetti Hi,
Were you able to get this onnx model running for custom object ?
